hookman/__main__.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+"""Console script for hookman."""
```

Comment:
You need to update the `entry_point` on `setup.py` as well ;)
```python
    entry_points={
        'console_scripts': [
            'hookman=hookman.__main__:cli',
        ],
    },
```

Reply:
Good catch!